### DISCUSSION TITLE
[question] build next.js project locally & deploy

### DISCUSSION THREAD

**Discussion by skene** [vercel/vercel#4451] [Help] [answered]
Basically the title says it all, is it possible to build my Next.js project locally, and deploy to Vercel while still supporting Next.js hybrid rendering (static/ssr) functionality. I've attempted to override the Build Command to be empty, and set the Output Directory appropriately; which leads to the output directory being hosted as a static file directory. 

Thanks in advanced.

**Comment by mcsdevv** [comment, marked as answer]
Hello @skene, thanks for raising this. Next.js projects must be built by Vercel, else they will miss the information required for routing.
